data/human/campaign events.txt

```diff
@@ -1251,10 +1251,10 @@ event "Thule becomes independent"
 		fleet "Large Free Worlds" 1500
 	planet "Thule"
 		add attributes "ex-pirate"
-		description `Thule is a mountainous world with a population of nearly a billion, settled in the early days of space exploration. When the colony was first established, the Earth government did not yet have the strength to control any system but its own, and when the Republic was formed, Thule did not join, and has repulsed all efforts to force them to do so.`
+		description `Thule is a mountainous world with a population of nearly a billion, settled in the early days of space exploration. When the colony was first established, the government on Earth did not yet have the strength to control any system but its own, and when the Republic was formed, Thule did not join, and has repulsed all efforts to force them to do so.`
```

Comment:
Hmmm, the original here was a bit wordy, and the edits don't help much. Maybe:
```suggestion
		description `Thule is a mountainous world with a population of nearly a billion, settled in the early days of space exploration. When the colony was first established, the government on Earth did not yet have the strength to control any system but its own. Likewise, Thule refused to join the Republic when it was formed and has repulsed all efforts to force them to do so.`
```

Reply:
I think "likewise" implies a similar action more than a consequence. "As such" may be better:

`As such, Thule refused to join the Republic when it was formed, and has since repulsed all efforts to force them to do so.`